dask/array/utils.py

```diff
@@ -565,6 +565,22 @@ def svd_flip(u, v, u_based_decision=False):
     return u, v
 
 
+################
+# scipy.linalg #
+################
```

Comment:
No other functions in this file have this comment, it makes me think `_is_nep18_active` is part of `scipy.linalg`. I would remove this.